### ISSUE TITLE
Cannot export transaction from localhost network (hardhat node) in hardhat project

### ISSUE THREAD

**Issue by CyclopsRex** [Tenderly/tenderly-cli#77]
Running `tenderly export <trx hash>` in my hardhat project fails.

Here's the end of the output from `tenderly export <trx hash> --output json --debug`:

```{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(*Logger).Log","level":"info","msg":"Collecting network information...\n","time":"2022-04-05T23:38:37+01:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(*Logger).Log","level":"info","msg":"Collecting transaction information...\n","time":"2022-04-05T23:38:37+01:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(*Logger).Log","level":"info","msg":"Collecting contracts...","time":"2022-04-05T23:38:37+01:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:314","func":"github.com/sirupsen/logrus.(*Entry).Logf","level":"debug","msg":"Trying Hardhat config path: /Users/my_project_path/hardhat.config.js","time":"2022-04-05T23:38:37+01:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:314","func":"github.com/sirupsen/logrus.(*Entry).Logf","level":"debug","msg":"Trying Hardhat config path: /Users/my_project_path/hardhat.config.ts","time":"2022-04-05T23:38:37+01:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(*Logger).Log","level":"debug","msg":"encountered unexcepted error: runtime error: invalid memory address or nil pointer dereference","time":"2022-04-05T23:38:39+01:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(*Logger).Log","level":"error","msg":"\nEncountered an unexpected error. This can happen if you are running an older version of the Tenderly CLI.","time":"2022-04-05T23:38:39+01:00"}
```

And here is my (stripped-down) `hardhat.config.ts`:

```
import { HardhatUserConfig } from "hardhat/config";

const config: HardhatUserConfig = {
  solidity: {
    compilers: [
      {
        version: "0.8.9",
      },
    ],
  },
  networks: {
    localhost: {
      url: "http://127.0.0.1:8545",
    },
  },
  paths: {
    sources: "./contracts",
    tests: "./test",
    cache: "./cache",
    artifacts: "./artifacts",
  },
};

export default config;

```

`npx hardhat --version`: `2.8.0`
`tenderly version`: `Current CLI version: v1.4.1`

Any idea what's going wrong, or how I find out?

**Comment by shubbard76** [comment]
Same problem here (well same result anyway - - underlying issue seems to be 'runtime error: index out of range [0] with length 0').   I've been using tenderly without issue for ages but now this.  Any update from support given this is doesn't seem to be an isolated issues and breaks the core functionality of the platform.  I was on an earlier version so thought the error msg would be resolve by simply following the instructions in the error and upgrading to V 1.4.1 but alas not.  Upgraded and still same issue.  Tried completing reinitialising tenderly and hardhat but to no avail.  

Full debug info below: 

logrus.(Logger).Log","level":"info","msg":"Collecting network information...\n","time":"2022-04-11T12:07:13+07:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(Logger).Log","level":"info","msg":"Collecting transaction information...\n","time":"2022-04-11T12:07:13+07:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(Logger).Log","level":"info","msg":"Collecting contracts...","time":"2022-04-11T12:07:13+07:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:314","func":"github.com/sirupsen/logrus.(Entry).Logf","level":"debug","msg":"Trying Hardhat config path:/hh/hardhat.config.js","time":"2022-04-11T12:07:13+07:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(Logger).Log","level":"debug","msg":"encountered unexcepted error: runtime error: index out of range [0] with length 0","time":"2022-04-11T12:07:13+07:00"}
{"file":"/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:192","func":"github.com/sirupsen/logrus.(Logger).Log","level":"error","msg":"\nEncountered an unexpected error. This can happen if you are running an older version of the Tenderly CLI.","time":"2022-04-11T12:07:13+07:00"}



Any ETA for a fix on this pls?

**Comment by shubbard76** [comment]
Got around the problem by rolling back hardhat to 2.7.1

**Comment by IgorPetkovic** [comment]
@shubbard76  The issue should be resolved in the latest version ([1.4.2](https://github.com/Tenderly/tenderly-cli/releases/tag/v1.4.2)). Would you mind checking if everything is working as expected now?

**Comment by shubbard76** [comment]
Works perfectly now... thanks very much :)
